dom.bs

```diff
@@ -1028,6 +1028,22 @@ specified otherwise it returns null.
 <p class="note no-backref"><a>Nodes</a>, <a for=/>shadow roots</a>, and <a>documents</a> override
 the <a>get the parent</a> algorithm.
 
+<p>Each {{EventTarget}} object can have an associated
+<dfn export for=EventTarget>activation behavior</dfn> algorithm. The <a>activation behavior</a>
```

Comment:
Should the concept be dropped from HTML in favor of this? It becomes a bit weird to have activation behavior for both elements and event targets.